README_GR.md

```diff
@@ -0,0 +1,148 @@
+# Eliza 🤖
+
+<div align="center">
+  <img src="./docs/static/img/eliza_banner.jpg" alt="Eliza Banner" width="100%" />
+</div>
+
+<div align="center">
+
+📖 [Τεκμηρίωση](https://elizaos.github.io/eliza/) | 🎯 [Παραδείγματα](https://github.com/thejoven/awesome-eliza)
+
+</div>
+
+## 🌍 Μεταφράσεις README
+[中文说明](./README_CN.md) | [日本語の説明](./README_JA.md) | [한국어 설명](./README_KOR.md) | [Persian](./README_FA.md) | [Français](./README_FR.md) | [Português](./README_PTBR.md) | [Türkçe](./README_TR.md) | [Русский](./README_RU.md) | [Español](./README_ES.md) | [Italiano](./README_IT.md) | [ไทย](./README_TH.md) | [Deutsch](./README_DE.md) | [Tiếng Việt](./README_VI.md) | [עִברִית](https://github.com/elizaos/Elisa/blob/main/README_HE.md) | [Tagalog](./README_TG.md) | [Polski](./README_PL.md) | [Arabic](./README_AR.md) | [Hungarian](./README_HU.md) | [Srpski](./README_RS.md) | [Română](./README_RO.md) | [Nederlands](./README_NL.md) | [Ελληνικά](./README_GR.md)
+
+## 🚩 Επισκόπηση
+<div align="center">
+  <img src="./docs/static/img/eliza_diagram.png" alt="Eliza Diagram" width="100%" />
+</div>
+
+## ✨ Χαρακτηριστικά
+
+- 🛠️ Πλήρεις συνδέσεις για Discord, Twitter και Telegram
+- 🔗 Υποστήριξη για κάθε μοντέλο (Llama, Grok, OpenAI, Anthropic, κ.λπ.)
+- 👥 Υποστήριξη πολλών πρακτόρων και δωματίων
+- 📚 Εύκολη ενσωμάτωση και αλληλεπίδραση με τα έγγραφά σας
+- 💾 Ανακτήσιμη μνήμη και αποθήκευση εγγράφων
+- 🚀 Εξαιρετικά επεκτάσιμο - δημιουργήστε τις δικές σας δράσεις και πελάτες
+- ☁️ Υποστήριξη για πολλά μοντέλα (τοπικά Llama, OpenAI, Anthropic, Groq, κ.λπ.)
+- 📦 Έτοιμο για χρήση!
+
+[Σχολείο για προγραμματιστές για Πράκτορες Τεχνητής Νοημοσύνης (ΑΙ)](https://www.youtube.com/watch?v=ArptLpQiKfI&list=PLx5pnFXdPTRzWla0RaOxALTSTnVq53fKL)
+
+## 🎯 Περιτπώσεις για χρήση
+
+- 🤖 Chatbots
+- 🕵️ Αυτόνομοι πράκτορες
+- 📈 Διαχείριση επιχειρηματικών διαδικασιών
+- 🎮 NPC σε βιντεοπαιχνίδια
+- 🧠 Trading
+- 🚀 Γρήγορη Εκκίνηση
+
+
+## 🚀 Γρήγορη Εκκίνηση
+
+## Προαπαιτούμενα
+
+- [Python 2.7+](https://www.python.org/downloads/)
+- [Node.js 23+](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [pnpm](https://pnpm.io/installation)
+- 
```

Comment:
<img width="354" alt="Screenshot 2025-01-12 at 7 48 08 AM" src="https://github.com/user-attachments/assets/eefd4022-b511-4c4d-b983-94e2a088e387" />

Reply:
thank you, updated. @tcm390